3rd-party-scripts/SimpleHTTPSServer.java

```diff
@@ -0,0 +1,105 @@
+import java.io.*;
+import java.net.InetSocketAddress;
+import java.lang.*;
+import java.net.URL;
```

Comment:
There are a number of unused imports here; I don't see `java.net.URL`, `java.io.InputStreamReader`, `java.io.Reader`, `java.net.URLConnection`, `javax.net.ssl.HostnameVerifier`, `javax.net.ssl.HttpsURLConnection`, `javax.net.ssl.SSLSession`, `javax.net.ssl.TrustManager`, `javax.net.ssl.X509TrustManager`, `java.security.cert.X509Certificate`,`java.net.InetAddress`, or `com.sun.net.httpserver.HttpServer` used.

After fixing my other comments, `com.sun.net.httpserver.HttpsExchange` can probably also be dropped.

Reply:
yes, removed all unused